kernel/README.md

```diff
@@ -1,9 +1,25 @@
+* [Build Kata Containers Kernel](#build-kata-containers-kernel)
+    * [Requirements](#requirements)
+    * [Setup kernel source code](#setup-kernel-source-code)
+* [Build the kernel](#build-the-kernel)
+    * [Install the Kernel in the default path for Kata](#install-the-kernel-in-the-default-path-for-kata)
+    * [Submit Kernel Changes](#submit-kernel-changes)
+    * [How is it tested.](#how-is-it-tested)
```

Comment:
Nit: period in heading.

Reply:
heh, guess you'd like me to fix that as well? :-) The auto toc generator put it in... ;-)

Reply:
I know it's technically outside the scope of this PR, but maybe one tiny extra commit to remove the period in the heading before your commit...?  :)

Reply:
See if the '.' is done now...